package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "start": "yarn build && yarn serve",
     "build": "yarn build-scss && yarn build-js && yarn build-class-references",
-    "build-tokens": "((style-dictionary build --verbose --config ./sd.config.json) || true)",
+    "build-tokens": "node ./sd.config.mjs",
```

Comment:
You would also have to change this in `main` before this can be built on Percy, using the `|| true` trick in the first PR, then reverting it to this in this PR.

So, first PR is:
```
build-tokens": "((node ./sd.config.mjs) || true)",
```

Then this PR can remain as-is:
```
"build-tokens": "node ./sd.config.mjs",
```